.circleci/config.yml

```diff
@@ -0,0 +1,101 @@
+version: 2.1
+commands:
+  restore-cache:
+    steps:
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package-lock.json" }}
+            - v1-dependencies-
+  install-dependencies:
+    steps: 
+      - run: npm install
+  save-cache:
+    steps:
+      - save_cache:
+          key: v1-dependencies-{{ checksum "package-lock.json" }}
```

Comment:
in most cases it's a good idead to add a way to manually invalidate the cache.
try to use a timestamp from projects envs in cache key